metron-platform/metron-indexing/src/main/scripts/start_hdfs_topology.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+# 
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+# 
+#     http://www.apache.org/licenses/LICENSE-2.0
+# 
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+METRON_VERSION=${project.version}
+METRON_HOME=/usr/metron/$METRON_VERSION
+TOPOLOGY_JAR=metron-elasticsearch-$METRON_VERSION-uber.jar
+storm jar $METRON_HOME/lib/$TOPOLOGY_JAR org.apache.storm.flux.Flux --remote $METRON_HOME/flux/indexing/batch/remote.yaml --filter $METRON_HOME/config/hdfs.properties
```

Comment:
In the MPack we refer to this as the "batch" topology.  But in this script we refer to it as the "HDFS" topology.  We should probably stick with one nomenclature.

Reply:
I almost think that if it is going to be hdfs, we should call it hdfs.  If we have multiple topologies based on the batch indexer, each will end up with a unique name anyways ( S3 indexer etc ).

Reply:
Very true.  I think you're right, @ottobackwards

Reply:
Yeah, I prefer to keep the scripts `hdfs` and `elasticsearch`